pages/phases/certificate.md

```diff
@@ -20,16 +20,27 @@ roles: [hr, hiring-mgr]
   {% include toolkit/certificate.html %}
 </div>
 
-## HR Preference Adjudication
+<div class="usa-alert usa-alert--info" >
+  <div class="usa-alert__body">
+    <h3 class="usa-alert__heading">Do not conduct a separate minimum qualification review</h3>
+    <p class="usa-alert__text">
+      After SMEs have completed interviews and identified the applicants who are found to meet qualifications, do not conduct a separate minimum qualification review. For more information, see "<a href="{{ site.baseurl }}/about/differences/#sme-assessment">How our process is different</a>."
```

Comment:
this needs to explain a little more - like we're not blinding trusting SMEs here- HR already reviewed all the justification for any cuts throughout the process. as long as those were justified throughout then at this point there is no need for HR to redo a review of technical requirements. HR can do a review of pay settings to make sure any federal employees qualify for the certificate.

Reply:
could incorporate some of the language from Ana to Kathy